src/sidebar/tabcenter.css

```diff
@@ -643,43 +645,19 @@ body[platform="win"] #searchbox-input:placeholder-shown {
 }
 
 .tab-host {
-    color: var(--secondary-text-color);
+    opacity: 0.5;
 }
 
 #tablist-wrapper.shrinked .tab-host {
     display: none;
 }
 
-.tab-title-wrapper::after {
-    content: '';
-    position: absolute;
-    width: 60px;
-    top: 0;
-    bottom: 0;
-    right: 0;
-    --tab-background: var(--tab-background-normal);
-    background-image: linear-gradient(to right, hsla(var(--tab-background), 0), hsl(var(--tab-background)) 30%);
-    transform: translateX(36px);
-}
-
-.tab.pinned > .tab-title-wrapper::after {
-    --tab-background: var(--tab-background-pinned);
-}
-
-.tab.active > .tab-title-wrapper::after {
-    --tab-background: var(--tab-background-active);
-}
-
-.tab:not(.active):hover > .tab-title-wrapper::after {
-    --tab-background: var(--tab-background-hover);
-}
-
-.tab:hover > .tab-title-wrapper::after {
-    transform: translateX(0);
+.tab-title-wrapper {
+    mask-image: linear-gradient(to left, transparent 0, black 30%);
```

Comment:
I find 30% to be a bit too wide, how about we use the `2em` Firefox uses natively?